common/model/global-alert.js

```diff
@@ -0,0 +1,4 @@
+export type GlobalAlert = {|
+  text: string,
```

Comment:
As this is going to contain HTML - how do we feel about setting it as a `HTMLString` and then choosing how to render it only at the edge of the app i.e. rendering?

https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/services/prismic/types.js#L45

Reply:
if the text is being `deP`-ed (so we end up with e.g. `words without opening tags <i>but maybe</i> <strong>some tags</strong> in the middle`, does that still count as an `HTMLString`? My guess is not.

Reply:
No - but that function is absolute nonsense, hence why we started having using these classes, which feel a little more ✨ 

https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/styles/utilities/_root-scope-classes.scss#L408-L416

Reply:
Got it.